stdlib/std/cells.tact

```diff
@@ -147,4 +157,19 @@ inline fun emptyCell(): Cell {
 
 inline fun emptySlice(): Slice {
     return emptyCell().asSlice();
-}
\ No newline at end of file
+}
+
+asm fun stringHash(data: Slice): Int { SHA256U }
```

Comment:
This one doesn't make much sense, since in Tact `String` and `Slice` are considered different types, while the `Slice` already has the `.hash()` method which uses the same TVM instruction. And for both that method and this function, the user would have to do explicit conversion `theirString.asSlice()`, so this function `stringHash()` is probably redundant and should be removed :)

Reply:
yes, theres `.hash` method for `String` type as well. Removed the function.